src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Colors {
+    Red,
+    White,
+    Black,
+    Pink,
+    Purple,
+    Green,
+    Yellow,
+    Grey

```

Comment:
let's write them in upper case
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names